clients/da-vinci-client/src/main/java/com/linkedin/davinci/storage/StorageService.java

```diff
@@ -207,6 +223,10 @@ private void restoreAllStores(
             if (ExceptionUtils.recursiveClassEquals(e, RocksDBException.class)) {
               LOGGER.error("Could not load the following store : " + storeName, e);
               aggVersionedStorageEngineStats.recordRocksDBOpenFailure(storeName);
```

Comment:
I think we should only record this metric if the Server receives an exception and can't delete it.
And we might need to put up an alert against it since it is critical.
But the storage engine, which can be deleted, I guess a warn log should be good enough.